includes/fields/class-evf-field-number.php

```diff
@@ -36,13 +36,89 @@ public function __construct() {
 					'placeholder',
 					'label_hide',
 					'css',
+					'maximum_number',
+					'minimum_number',
+
 				),
 			),
 		);
 
 		parent::__construct();
 	}
 
+
+
+
+	/**
+	 * Minimum number field option
+	 *
+	 * @since      1.4.5
+	 *
+	 * @param array $field
+	 */
+	public function minimum_number( $field ) {
+		$lbl  = $this->field_element(
+			'label',
+			$field,
+			array(
+				'slug'    => 'minimum_number',
+				'value'   => esc_html__( 'Minimum Number', 'everest-forms-pro' ),
+				'tooltip' => sprintf( esc_html__( 'Enter Minimum Number Allowed.', 'everest-forms-pro' ) ),
+			),
+			false
+		);
+		$fld  = $this->field_element(
+			'text',
+			$field,
+			array(
+				'slug'  => 'minimum_number',
+				'value' => trim( $field['minimum_number'] ) !== null ? $field['minimum_number'] : '',
+			),
+			false
+		);
+		$args = array(
+			'slug'    => 'minimum_number',
+			'content' => $lbl . $fld,
+		);
+		$this->field_element( 'row', $field, $args );
+	}
+
+
+	/**
+	 * Maximum number field option
+	 *
+	 * @since      1.4.5
+	 *
+	 * @param array $field
+	 */
+	public function maximum_number( $field ) {
+		$lbl  = $this->field_element(
+			'label',
+			$field,
+			array(
+				'slug'    => 'maximum_number',
+				'value'   => esc_html__( 'Maximum Number', 'everest-forms-pro' ),
+				'tooltip' => sprintf( esc_html__( 'Enter Maximum Number Allowed. ', 'everest-forms-pro' ) ),
```

Comment:
This sentence should be clear and appropriate so that users can understand easily.